src/style.css

```diff
@@ -0,0 +1,109 @@
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --color: #3d4e61;
+}
+
+html {
+  padding: 0;
+  font-family: "Avenir", sans-serif;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  font-weight: 300;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-weight: 900;
+}
+
+.form-big {
+  height: 70px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background-color: #fff;
+}
+
+.form-small {
+  height: 42px;
+  border-radius: 3px;
+  border: 1px solid #e1e7ed;
+  background-color: #fff;
```

Comment:
Consider using variables for repeating colors

Reply:
agree, please do it